source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -170,6 +170,73 @@ The JWT contains the following claims:
 
+### Validate the core identity claim JWT using a public key
+
+To validate the core identity claim JWT, you must use a public key. GOV.UK One Login publishes the public keys in a [Decentralized Identifier (DID) document](https://www.w3.org/TR/did-core/). 
+
+<%= warning_text('GOV.UK One Login regularly rotates its public keys. You must [read the guidance on understanding GOV.UK One Login’s key rotation](#understand-govuk-one-logins-key-rotations) to make sure your application continues to work as expected.') %>
+
+This is an example of a web DID document published by GOV.UK One Login:
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/jwk/v1"
+  ],
+  "id": "did:web:identity.account.gov.uk",
+  "assertionMethod": [
+    {
+      "id": "b7863b6926193d93b48808cbabcbc8a414d0080f81c8779c0a54491551a35816",
+      "type": "JsonWebKey",
+      "controller": "did:web:identity.account.gov.uk",
+      "publicKeyJwk": {
+        "kty": "EC",
+        "crv": "P-256",
+        "x": "QrP65yghuglwPkEl11oMaabr4WqAMjuvztBYb7T4Ipo",
+        "y": "CSQNybYbCZLl-Xr1OA3pcxjC6qZrG7JPqwXgo-9fHLM"
+      }
+    }
+  ]
+}
+```
+
+#### Use the `kid` (key ID) to determine which public key signed the JWT
+
+When validating a JWT, the JWT header will include the `kid` (key ID). This will be either `did:web:identity.integration.account.gov.uk#{unique key ID}` for the Integration environment, or `did:web:identity.account.gov.uk#{unique key identifier} ` for the Production environment. 
+
+You need to use the `kid` to determine which public key from the DID document was used to sign the JWT – this is important in case GOV.UK One Login has rotated its public keys. If there are multiple keys visible in the DID document, GOV.UK One Login is in the process of rotating its keys.
+
+1. Split the `kid` from the JWT header into two parts: before the `#` and after the `#` – the first part is the controller ID and the second part is the unique key ID. For example, in the `kid` `did:web:identity.integration.account.gov.uk#c9f8da1c87525bb41653583c2d05274e85805ab7d0abc58376c7128129daa936`, the controller ID is `did:web:identity.integration.account.gov.uk` and the unique key ID is `c9f8da1c87525bb41653583c2d05274e85805ab7d0abc58376c7128129daa936`.
+1. Download the DID document from the DID endpoint you need:
+   * Integration: [https://identity.integration.account.gov.uk/.well-known/did.json](https://identity.integration.account.gov.uk/.well-known/did.json)
+   * Production: [https://identity.account.gov.uk/.well-known/did.json](https://identity.account.gov.uk/.well-known/did.json)
+1. Make sure the controller ID matches the `id` in the DID document.
+1. Find the key in `assertionMethods` that has an `id` field matching the unique key ID from the `kid` in the DID document – if a key with a matching `id` is not present, you should not trust the identity and should contact GOV.UK One Login to report an incident.
```

Comment:
This isn't quite right as `assertionMethods` is a list, not an object/map.

Reply:
It's a list of key objects. The word `key` here is referring to a key object in the list, rather map key (field/claim name).

If we add the word "object" it should make it clearer.